src/main/kotlin/io/github/enyason/predictions/ReplicateConfig.kt

```diff
@@ -0,0 +1,7 @@
+package io.github.enyason.predictions
+
+data class ReplicateConfig(
```

Comment:
can this class be made internal to the package where it is being used?
We don't want users calling it and changing the base_url

Reply:
It shouldn't be internal, the user should be able initialise the SDK using either the Config or just the API token. Additional fields can be introduced in the future which is why we should have a data class for all the configs

In my latest commits, I showed how the SDK or API can be initialised